build-tools/config/webpack/webpack.partial.conf.plugins.js

```diff
@@ -108,6 +107,22 @@ module.exports = ({ config, isDevelopment, buildType }) => webpackConfig => {
         }),
       );
     }
+    if (config.enablePrerender) {
+      plugins.push(
+        new PrerenderSPAPlugin({
+          staticDir: path.join(config.projectRoot, 'dist'),
+          // Add routes to prerender.
+          routes: ['/', '/about'],
```

Comment:
could this be automated?

Reply:
We can actually create a function that checks for the available locales and the available route names to create those automatically but I don't know if this should stay out of the out of the box configuration of the skeleton as you might need to pre render just certain pages.